src/linter.ts

```diff
@@ -13,7 +13,7 @@ import { execV } from "./exec";
 import { resolve, relative, dirname, join } from "path";
 import { readdirSync } from "fs";
 
-const outDir = join(tmpdir(), "vscode_vlang");
+const outDir = join(tmpdir(), "vscode_vlang", "lint.c");
```

Comment:
The `outDir` variable contains no directory path, but a file path. It might be confusing.